phpBB/adm/style/acp_styles.html

```diff
@@ -32,6 +32,10 @@
 
 <!-- IF L_EXPLAIN --><p>{L_EXPLAIN}</p><!-- ENDIF -->
 
+<fieldset class="quick">
+	<span class="small"><a href="https://www.phpbb.com/customise/db/styles-2" target="_blank">{L_BROWSE_STYLES_DATABASE}</a></span>
```

Comment:
International support teams:
http://www.phpbb.nl/downloads/vertalingen/
http://www.phpbb-fr.com/customise/db/
...

We have quite a lot of ISTs that run their own version of titania